main.css

```diff
@@ -99,11 +99,7 @@ body {
     color: #F57D19;
 }
 
-.scrolling-box {
-    height: 100vh;
-    overflow-y: scroll;
-    scroll-behavior: smooth;
-}
+/*TODO usunąłem scroll smootha*/
```

Comment:
To usuń też ten komentarz :)

Reply:
hehe :P wbrew pozorom ten TODO zawiera wiecej informacji niz jest tam napisane :) dlatego chwilowo go nie usuwam